CMakeLists.txt

```diff
@@ -128,23 +128,25 @@ include(cmake/SetupUmpireThirdParty.cmake)
 
 add_subdirectory(src)
 
-configure_file(
-  umpire-config.cmake.in
-  "${PROJECT_BINARY_DIR}/umpire-config.cmake" @ONLY)
+configure_package_config_file(
+  "${PROJECT_SOURCE_DIR}/umpire-config.cmake.in"
+  "${PROJECT_BINARY_DIR}/umpire-config.cmake"
+  PATH_VARS CMAKE_INSTALL_PREFIX
+  INSTALL_DESTINATION lib/cmake/umpire)
 
 install(FILES
-  "${PROJECT_BINARY_DIR}/umpire-config.cmake"
-  DESTINATION share/umpire/cmake)
+  ${PROJECT_BINARY_DIR}/umpire-config.cmake
+  DESTINATION lib/cmake/umpire)
 
 write_basic_package_version_file(
   ${PROJECT_BINARY_DIR}/umpire-config-version.cmake
   COMPATIBILITY SameMajorVersion)
 
 install(FILES 
   "${PROJECT_BINARY_DIR}/umpire-config-version.cmake"
-  DESTINATION share/umpire/cmake)
+  DESTINATION lib/cmake/umpire)
 
-install(EXPORT umpire-targets DESTINATION share/umpire/cmake)
+install(EXPORT umpire-targets DESTINATION lib/cmake/umpire)
```

Comment:
Why this move?  It is more conventional, but raja uses the share/<proj>/cmake convention, and both are supported.  Want to update this in all framework packages?

Actually... To that point, it might be worth adding something to BLT to make it trivial to produce and/or install one of these so BLT-based packages are consistent.

Reply:
I noticed that camp was using it, so thought it might be a better option. I can definitely revert the change, but we should have all project consistent.

Reply:
Ironically I was thinking I changed it in camp to match raja, but it looks like you're right.  Let's go with this since you've already got it in here and I'll make sure camp stays to match.  We can update raja as part of 1136.

Reply:
Sounds good! The only other thing of note is the use of `configure_package_config_file`, which is missing from the RAJA PR. I can go ahead and make the changes there if you think this looks right.